### ISSUE TITLE
Can't view trade confirmation to accept them anymore

### ISSUE THREAD

**Issue by restt** [dyc3/steamguard-cli#29]
Currently using the most recent version, 0.3.4.1.0

Unhandled Exception:
System.AggregateException: One or more errors occurred. ---> System.ArgumentOutOfRangeException: Specified argument was out of the range of valid values.
Parameter name: i
  at System.Text.RegularExpressions.MatchCollection.get_Item (System.Int32 i) [0x00015] in <50d80b08c1a5449282b22aedf03ce925>:0 
  at SteamAuth.SteamGuardAccount+<FetchConfirmationsAsync>c__async0.MoveNext () [0x001c3] in <e9380748a38f4f39b77c7f4ae9afdc4b>:0 
   --- End of inner exception stack trace ---
  at System.Threading.Tasks.Task.ThrowIfExceptional (System.Boolean includeTaskCanceledExceptions) [0x00011] in <62f5937022004555807e6c57c33f6684>:0 
  at System.Threading.Tasks.Task.Wait (System.Int32 millisecondsTimeout, System.Threading.CancellationToken cancellationToken) [0x00043] in <62f5937022004555807e6c57c33f6684>:0 
  at System.Threading.Tasks.Task.Wait () [0x00000] in <62f5937022004555807e6c57c33f6684>:0 
  at SteamGuard.Program.processConfirmations (SteamAuth.SteamGuardAccount account) [0x0007c] in <3cb839bf24f747bca86bd11fed201ffe>:0 
  at SteamGuard.Program.Trade (System.String user, System.String passkey) [0x000c5] in <3cb839bf24f747bca86bd11fed201ffe>:0 
  at SteamGuard.Program.Main (System.String[] args) [0x00430] in <3cb839bf24f747bca86bd11fed201ffe>:0 
---> (Inner Exception #0) System.ArgumentOutOfRangeException: Specified argument was out of the range of valid values.
Parameter name: i
  at System.Text.RegularExpressions.MatchCollection.get_Item (System.Int32 i) [0x00015] in <50d80b08c1a5449282b22aedf03ce925>:0 
  at SteamAuth.SteamGuardAccount+<FetchConfirmationsAsync>c__async0.MoveNext () [0x001c3] in <e9380748a38f4f39b77c7f4ae9afdc4b>:0 <---

[ERROR] FATAL UNHANDLED EXCEPTION: System.AggregateException: One or more errors occurred. ---> System.ArgumentOutOfRangeException: Specified argument was out of the range of valid values.
Parameter name: i
  at System.Text.RegularExpressions.MatchCollection.get_Item (System.Int32 i) [0x00015] in <50d80b08c1a5449282b22aedf03ce925>:0 
  at SteamAuth.SteamGuardAccount+<FetchConfirmationsAsync>c__async0.MoveNext () [0x001c3] in <e9380748a38f4f39b77c7f4ae9afdc4b>:0 
   --- End of inner exception stack trace ---
  at System.Threading.Tasks.Task.ThrowIfExceptional (System.Boolean includeTaskCanceledExceptions) [0x00011] in <62f5937022004555807e6c57c33f6684>:0 
  at System.Threading.Tasks.Task.Wait (System.Int32 millisecondsTimeout, System.Threading.CancellationToken cancellationToken) [0x00043] in <62f5937022004555807e6c57c33f6684>:0 
  at System.Threading.Tasks.Task.Wait () [0x00000] in <62f5937022004555807e6c57c33f6684>:0 
  at SteamGuard.Program.processConfirmations (SteamAuth.SteamGuardAccount account) [0x0007c] in <3cb839bf24f747bca86bd11fed201ffe>:0 
  at SteamGuard.Program.Trade (System.String user, System.String passkey) [0x000c5] in <3cb839bf24f747bca86bd11fed201ffe>:0 
  at SteamGuard.Program.Main (System.String[] args) [0x00430] in <3cb839bf24f747bca86bd11fed201ffe>:0 
---> (Inner Exception #0) System.ArgumentOutOfRangeException: Specified argument was out of the range of valid values.
Parameter name: i
  at System.Text.RegularExpressions.MatchCollection.get_Item (System.Int32 i) [0x00015] in <50d80b08c1a5449282b22aedf03ce925>:0 
  at SteamAuth.SteamGuardAccount+<FetchConfirmationsAsync>c__async0.MoveNext () [0x001c3] in <e9380748a38f4f39b77c7f4ae9afdc4b>:0 <---

Only happened recently, i could view my trades no problem like a week or so ago. Not sure what the problem is.

I'm using linux mint.

**Comment by melvyn2** [comment]
Same problem as #28

**Comment by melvyn2** [comment]
Sorry, it's actually a different exception, but with similar symptoms.

**Comment by restt** [comment]
Yup, my auth is fixed and now i can confirm trades as normal. Much appreciate it.
